scripts/index.ts

```diff
@@ -1,6 +1,9 @@
 import { join } from 'path';
+import * as url from 'url';
 
-import * as build from './build';
+import * as build from './build.js';
+
+const __dirname = url.fileURLToPath(new URL('.', import.meta.url));
```

Comment:
found this on some blog post as an ESM-compatible polyfill for the `__dirname` global that is normally set when running a commonjs module